packages/ui-components/src/components/Map/Legend/LegendEntry.js

```diff
@@ -3,18 +3,26 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import PropTypes from 'prop-types';
 import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import MuiButton from '@material-ui/core/Button';
 
-const Key = styled.div`
+const Button = styled(MuiButton)`
   display: flex;
   flex-direction: row;
   align-items: center;
-  padding: 0.3rem;
+  padding: 0.2rem 0.3rem;
+  font-weight: 400;
+  letter-spacing: 0;
   pointer-events: auto;
+  margin-bottom: 0.1rem;
   cursor: pointer;
-  ${p => (p.hidden ? 'opacity: 0.5;' : '')}
+  opacity: ${props => (props.hidden ? '0.5' : '1')};
```

Comment:
Good improvement going for `props`, but it does make me wonder why `hidden` goes to `0.5` rather than `0` opacity

Reply:
perhaps hidden is a bad name. The map overlay markers that are hidden and the legend button is 'greyed out' with 50% opacity.

Reply:
Aah yeah that makes sense, the prop name relates more to the thing the legend entry controls than the legend entry itself. I'm fine to leave it because I can't think of anything better that captures the idea across both the legend entry and the effect it's having